api/Element.json

```diff
@@ -5789,7 +5789,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
I've filed https://github.com/mdn/browser-compat-data/issues/6683 about api.Element.name not being a useful table. This is correct, though.